README.md

```diff
@@ -2,4 +2,37 @@
 
 ![](https://github.com/qiime2/q2-composition/workflows/ci/badge.svg)
 
-This is a QIIME 2 plugin. For details on QIIME 2, see https://qiime2.org.
\ No newline at end of file
+This is a QIIME 2 plugin. For details on QIIME 2, see https://qiime2.org.
+
+## Install Guide
+
+This plugin is a wrapper for the ANCOMBC package. We are working with the
+[RELEASE_3_15](https://github.com/FrederickHuangLin/ANCOMBC/tree/RELEASE_3_15)
+branch, as this is where the functionality for ANCOM, ANCOM-BC, and SECOM are
+all located. This release hasn't been published on bioconductor yet, so the
+steps to install this latest release (for development purposes) are as follows:
+
+### Install R
+
+If you do not currently have R installed, you will need to do that prior to
+installing ANCOMBC. The appropriate CRAN mirror for your machine can be found
+[here](https://cran.r-project.org/mirrors.html).
```

Comment:
We want to use the R already in our conda environment (i.e. the last bioconductor release)